src/core/Geometry.js

```diff
@@ -260,6 +263,65 @@ Geometry.prototype = Object.assign( Object.create( EventDispatcher.prototype ),
 
 			}
 
+		}
+		if ( skinIndices !== undefined ) {
+
+			scope.skinIndices.push( new Vector4( skinIndices[ k ], skinIndices[ k + 1 ], skinIndices[ k + 2 ], skinIndices[ k + 3 ] ) );
+
+		}
+
+		if ( skinWeights !== undefined ) {
+
+			scope.skinWeights.push( new Vector4( skinWeights[ k ], skinWeights[ k + 1 ], skinWeights[ k + 2 ], skinWeights[ k + 3 ] ) );
+
+		}
+
+		var morphAttributes = geometry.morphAttributes;
+
+		if ( morphAttributes.position ) {
+
+			for ( const morphTargetAttribute of morphAttributes.position ) {
```

Comment:
Please don't use `const` and `let`. Also avoid `for of` loops.

Reply:
Ok